### PR TITLE
Move utils-module and filters into misc

### PR DESCRIPTION
**Pull request by pfirpfel**

### CODE REVIEW

File: contribs/gmf/src/mobile/measure/lengthComponent.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ goog.provide('gmf.mobile.measure.lengthComponent');
 
 goog.require('gmf');
 /** @suppress {extraRequire} */
```
Comment (sbrunner) [inline contribs/gmf/src/mobile/measure/lengthComponent.js]: To be removed

File: examples/measure.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ goog.require('ngeo.map.module');
 goog.require('ngeo.misc.btnComponent');
 goog.require('ngeo.misc.decorate');
 /** @suppress {extraRequire} */
```
Comment (sbrunner) [inline examples/measure.js]: same

File: examples/scaleselector.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ goog.provide('app.scaleselector');
 // webpack: import './scaleselector.css';
 // webpack: import './common_dependencies.js';
 /** @suppress {extraRequire} */
```
Comment (sbrunner) [inline examples/scaleselector.js]: also

File: src/editing/createfeaturecomponent.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ goog.provide('ngeo.editing.createfeatureComponent');
 goog.require('goog.asserts');
 goog.require('ngeo');
 /** @suppress {extraRequire} */
```
Comment (sbrunner) [inline src/editing/createfeaturecomponent.js]: more :-)

File: src/grid/component.js [hunk 5]
```diff
@@ -3,7 +3,7 @@ goog.provide('ngeo.grid.component');
 goog.require('goog.asserts');
 goog.require('ngeo');
 /** @suppress {extraRequire} */
```
Comment (sbrunner) [inline src/grid/component.js]: :-)

File: src/measure/azimut.js [hunk 6]
```diff
@@ -3,7 +3,7 @@ goog.provide('ngeo.measure.azimut');
 goog.require('ngeo');
 goog.require('ngeo.draw.Controller');
 /** @suppress {extraRequire} */
```
Comment (sbrunner) [inline src/measure/azimut.js]: +1

File: src/measure/length.js [hunk 7]
```diff
@@ -3,7 +3,7 @@ goog.provide('ngeo.measure.length');
 goog.require('ngeo');
 goog.require('ngeo.draw.Controller');
 /** @suppress {extraRequire} */
```
Comment (sbrunner) [inline src/measure/length.js]: ...

File: src/misc/FeatureHelper.js [hunk 8]
```diff
@@ -3,7 +3,7 @@ goog.provide('ngeo.misc.FeatureHelper');
 goog.require('goog.asserts');
 goog.require('ngeo');
 /** @suppress {extraRequire} */
```
Comment (sbrunner) [inline src/misc/FeatureHelper.js]: more

File: contribs/gmf/src/mobile/measure/lengthComponent.js [hunk 9]
```diff
@@ -13,7 +13,9 @@ goog.require('ol.style.RegularShape');
 goog.require('ol.style.Stroke');
 goog.require('ol.style.Style');
 
-gmf.mobile.measure.lengthComponent = angular.module('', []);
+gmf.mobile.measure.lengthComponent = angular.module('', [
```
Comment (ger-benjamin) [inline contribs/gmf/src/mobile/measure/lengthComponent.js]: ? no name ? I I pretty sure we need one !

File: contribs/gmf/src/map/mousepositionComponent.js [hunk 10]
```diff
@@ -4,7 +4,7 @@ goog.require('gmf');
 goog.require('goog.asserts');
 goog.require('ngeo');
 /** @suppress {extraRequire} */
```
Comment (sbrunner) [inline contribs/gmf/src/map/mousepositionComponent.js]: To be removed

File: contribs/gmf/src/datasource/ExternalDataSourcesManager.js [hunk 11]
```diff
@@ -6,7 +6,7 @@ goog.provide('gmf.datasource.ExternalDataSourcesManager');
 goog.require('gmf');
 /** @suppress {extraRequire} */
```
Comment (sbrunner) [inline contribs/gmf/src/datasource/ExternalDataSourcesManager.js]: To be removed

File: contribs/gmf/examples/contextualdata.js [hunk 12]
```diff
@@ -8,7 +8,7 @@ goog.require('gmf.contextualdata.module');
 /** @suppress {extraRequire} */
 goog.require('gmf.map.component');
 /** @suppress {extraRequire} */
```
Comment (sbrunner) [inline contribs/gmf/examples/contextualdata.js]: To be removed

File: src/misc/filters.js [hunk 13]
```diff
@@ -1,9 +1,13 @@
-goog.provide('ngeo.filters');
+goog.provide('ngeo.misc.filters');
 
 goog.require('ngeo');
 goog.require('ol.math');
 goog.require('ol.string');
 
+ngeo.misc.filters = angular.module('ngeoAngularFilters', []);
+
+ngeo.module.requires.push(ngeo.misc.filters.name);
```
Comment (ger-benjamin) [inline src/misc/filters.js]: You add it here, and remove it later ?

Comment (sbrunner) [reply src/misc/filters.js]: :-)

File: contribs/gmf/src/mobile/measure/lengthComponent.js (hunk 9 above)
Comment (sbrunner) [reply contribs/gmf/src/mobile/measure/lengthComponent.js]: Relay missing ...

File: src/misc/filters.js (hunk 13 above)
Comment (pfirpfel) [reply src/misc/filters.js]: Yes, I remove these ngeo.module.requires.push in the other PR.

I will rebase and remove lines later in #3455

Comment (ger-benjamin) [reply src/misc/filters.js]: OK

**Pull request merged**